### PR TITLE
Support Forge clients

### PR DESCRIPTION
**Pull request by GDWR**
Handle forge connections

### CODE REVIEW

File: flake.lock
Comment (fisher60) [inline flake.lock:1]: trash

File: .envrc
Comment (fisher60) [inline .envrc:1]: garbage

File: shell.nix
Comment (fisher60) [inline shell.nix:1]: Belongs in the bin

**Review by fisher60 — APPROVED** [review]

File: flake.lock
Comment (GDWR) [reply flake.lock:1]: image

File: .envrc
Comment (GDWR) [reply .envrc:1]: image

File: shell.nix
Comment (GDWR) [reply shell.nix:1]: image

**Pull request merged**